src/Shared/Microsoft.AspNetCore.Razor.Test.Common/Language/IntegrationTests/IntegrationTestBase.cs

```diff
@@ -34,6 +35,8 @@ static IntegrationTestBase()
         var referenceAssemblyRoots = new[]
         {
             typeof(System.Runtime.AssemblyTargetedPatchBandAttribute).Assembly, // System.Runtime
+            typeof(System.Console).Assembly,
+            typeof(Microsoft.AspNetCore.Razor.Hosting.RazorCompiledItemMetadataAttribute).Assembly, // Microsoft.AspNetCore.Razor shims
```

Comment:
Should we try switching over to `Basic.Reference.Assemblies` now that Jared has helpfully added the ASP.NET references?

Reply:
Yes, but not sure if it would be good to do in context of this PR; this pattern is used over multiple other places which should all get ideally switched over.

Reply:
In any case, I guess we should have a tracking issue. I filed https://github.com/dotnet/razor/issues/10343.